### ISSUE TITLE
Mediawiki run can't start with wrk

### ISSUE THREAD

**Issue by gaborkertesz** [facebookresearch/DCPerf#65]
Load generator is not found with recent wrk change.

I got the following error on x86 and Arm as well:

```
sudo ./benchpress_cli.py run oss_performance_mediawiki_mlp
Will run 1 job(s)
Running "oss_performance_mediawiki_mlp": Tuned +MLP run for oss_performance_mediawiki
Job execution command: ['./packages/mediawiki/run.sh', '-r/usr/local/hphpi/legacy/bin/hhvm', '-nnginx', '-L', 'wrk', '-s', 'benchmarks/oss_performance_mediawiki/wrk/wrk', '-R0', '-c0', '--', '--mediawiki-mlp', '--client-duration=600', '--client-timeout=11m', '--run-as-root', '--i-am-not-benchmarking']
stdout:
        Specified load generator  is not an executable or does not exist.
stderr:

Job "./packages/mediawiki/run.sh -r/usr/local/hphpi/legacy/bin/hhvm -nnginx -L wrk -s benchmarks/oss_performance_mediawiki/wrk/wrk -R0 -c0 -- --mediawiki-mlp --client-duration=600 --client-timeout=11m --run-as-root --i-am-not-benchmarking" returned non-zero exit status 1
```

Also the install step failed if 'unzip' is not installed:
```
+ pushd wrk
/home/ubuntu/DCPerf/benchmarks/oss_performance_mediawiki/wrk /home/ubuntu/DCPerf/benchmarks/oss_performance_mediawiki /home/ubuntu/DCPerf
+ git apply --check /home/ubuntu/DCPerf/packages/mediawiki/0004-wrk.diff
+ git apply /home/ubuntu/DCPerf/packages/mediawiki/0004-wrk.diff
+ make
echo LuaJIT-2.1
LuaJIT-2.1
make: unzip: No such file or directory
make: *** [Makefile:82: obj/LuaJIT-2.1] Error 127
Traceback (most recent call last):
  File "/home/ubuntu/DCPerf/./benchpress_cli.py", line 16, in <module>
    main()
  File "/home/ubuntu/DCPerf/benchpress/cli/main.py", line 354, in main
    args.command.run(args, bp.jobs)
  File "/home/ubuntu/DCPerf/benchpress/cli/commands/install.py", line 75, in run
    install_benchmark(job.install_script, env=env)
  File "/home/ubuntu/DCPerf/benchpress/lib/util.py", line 89, in install_benchmark
    raise Exception(f"Failed to run '{cmd_str}'")
Exception: Failed to run 'bash -x ./packages/mediawiki/install_oss_performance_mediawiki.sh'
``` 

Setting back to the following git commit fixed the problem: f893dac287a407ad3ba3b6cdd515801a3449cf93

**Comment by excelle08** [comment]
Can you install zip and try installing mediawiki again?

**Comment by gaborkertesz** [comment]
Yes, sorry I think I wasn't clear. So installing unzip fixed the installation issue and that revealed the runtime error (first block of my comment).

**Comment by excelle08** [comment]
I pushed c09d7eb trying to fix this, please try re-installing Mediawiki and running again

**Comment by gaborkertesz** [comment]
It works as expected now!

**Comment by gaborkertesz** [comment]
I suppose this should be updated as well in order to get score as the reported result is changed:
https://github.com/facebookresearch/DCPerf/blob/main/benchpress/lib/baseline.py#L40

``
  "job": "oss_performance_mediawiki_mlp",
  "metrics": {
    "Combined": {
      "Nginx 200": 89186,
      "Nginx 404": 3876,
      "Nginx P50 time": 0.047,
      "Nginx P90 time": 0.083,
      "Nginx P95 time": 0.095,
      "Nginx P99 time": 0.11,
      "Nginx avg bytes": 158875.50196643,
      "Nginx avg time": 0.050858825299263,
      "Nginx hits": 93062,
      "Wrk RPS": 155.07,
      "Wrk failed requests": 3876,
      "Wrk requests": 93055,
      "Wrk successful requests": 89179,
      "Wrk wall sec": 600.1,
      "canonical": 0
    }
``

**Comment by excelle08** [comment]
Thanks for catching this, I'll fix the score reporting for this and also for video transcode benchmark

**Comment by excelle08** [comment]
> I suppose this should be updated as well in order to get score as the reported result is changed from "Siege RPS" to "Wrk RPS". https://github.com/facebookresearch/DCPerf/blob/main/benchpress/lib/baseline.py#L40
> 
> ```
>   "job": "oss_performance_mediawiki_mlp",
>   "metrics": {
>     "Combined": {
>       "Nginx 200": 89186,
>       "Nginx 404": 3876,
>       "Nginx P50 time": 0.047,
>       "Nginx P90 time": 0.083,
>       "Nginx P95 time": 0.095,
>       "Nginx P99 time": 0.11,
>       "Nginx avg bytes": 158875.50196643,
>       "Nginx avg time": 0.050858825299263,
>       "Nginx hits": 93062,
>       "Wrk RPS": 155.07,
>       "Wrk failed requests": 3876,
>       "Wrk requests": 93055,
>       "Wrk successful requests": 89179,
>       "Wrk wall sec": 600.1,
>       "canonical": 0
>     }
> ```

Fixed in ce16488
